### DISCUSSION TITLE
Usage of ShardedJedisPool in a Non-Cluster Redis Environment in lastest version: Deprecated or Alternatives?

### DISCUSSION THREAD

**Discussion by RogerTaylor130** [xetorthio/jedis#3469] [Q&A] [answered]
Hi team,

I understand that ShardedJedisPool has been deprecated, and I would like to confirm if ShardedJedisPool is still available in any form in version 4.3.0. If not, how can I achieve the same functionality? Should I downgrade the Jedis version? Is that recommended? Or any Alternatives in the latest version?

Many thanks,
Roger.

**Comment by yangbodong22011** [comment, marked as answer]
Hello, @RogerTaylor130 you can use `JedisSharding` to replace `ShardedJedisPool` in Jedis 4.x.

**Reply by RogerTaylor130** [reply]
Hi @yangbodong22011. Thanks so much for your reply!
